oml/utils/misc_torch.py

```diff
@@ -58,6 +74,17 @@ def _check_is_sequence(val: Any) -> bool:
         return False
 
 
+def get_embedding_metric_from_callbacks(
+    callbacks: Collection[pl.Callback], key: str = f"{OVERALL_CATEGORIES_KEY}/cmc/1"
+) -> float:
+    for clb in callbacks:
+        if isinstance(clb, MetricValCallback):
```

Comment:
Using oml modules inside `oml.utils` can bring us to possible cyclic import(( Can you move it to Lightning-specific section?

Reply:
removed